Gemfile

```diff
@@ -1,11 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf',  '~> 3.2'
-gem 'chefspec',   '~> 4.2'
-gem 'foodcritic', '~> 4.0'
-gem 'rubocop',    '~> 0.30.1'
+gem 'berkshelf', '~> 4.0'
+gem 'chef-zero', '= 4.5'
```

Comment:
All started with issues of chef-zero 4.7. It is failing on ruby < 2.1. I was trying to find a common minimum version list.. 
... or we can get rid of ruby 2 from the list